gfootball/env/observation_processor.py

```diff
@@ -111,11 +158,14 @@ def get_frame(trace):
         field_coords=True,
         color=(0, 255, 0))
     letter = 'H'
+    # Goalkeeper
+    if player_idx == 0:
```

Comment:
Where player_idx comes from? Does it even compile?

Reply:
It comes from a few lines above, where we enumerate over all players